code/datums/brain_damage/severe.dm

```diff
@@ -330,3 +330,106 @@
 /datum/brain_trauma/severe/dyslexia/on_lose()
 	REMOVE_TRAIT(owner, TRAIT_ILLITERATE, TRAUMA_TRAIT)
 	..()
+
+/*
+ * Brain traumas that eldritch paintings apply
+ * This one is for "The Sister and He Who Wept" or /obj/structure/sign/painting/eldritch
+ */
+/datum/brain_trauma/severe/weeping
+	name = "The Weeping"
+	desc = "Patient hallucinates everyone as a figure called He Who Wept"
+	scan_desc = "H_E##%%%WEEP6%11S!!,)()"
+	gain_text = span_warning("HE WEEPS AND I WILL SEE HIM ONCE MORE")
+	lose_text = span_notice("You feel the tendrils of something slip from your mind.")
+	random_gain = FALSE
+	/// Our cooldown declare for causing hallucinations
+	COOLDOWN_DECLARE(weeping_hallucinations)
+
+/datum/brain_trauma/severe/weeping/on_life(seconds_per_tick, times_fired)
+	if(owner.stat != CONSCIOUS || owner.IsSleeping() || owner.IsUnconscious())
+		return
+	// If they have examined a painting recently
+	if(HAS_TRAIT(owner, TRAIT_ELDRITCH_PAINTING_EXAMINE))
+		return
+	if(!COOLDOWN_FINISHED(src, weeping_hallucinations))
+		return
+	owner.cause_hallucination(/datum/hallucination/delusion/preset/heretic, "Caused by The Weeping brain trauma")
+	owner.add_mood_event("eldritch_weeping", /datum/mood_event/eldritch_painting/weeping)
+	COOLDOWN_START(src, weeping_hallucinations, 10 SECONDS)
+	..()
+
+//This one is for "The First Desire" or /obj/structure/sign/painting/eldritch/desire
+/datum/brain_trauma/severe/flesh_desire
+	name = "The Desire for Flesh"
+	desc = "Patient seems to only be able to eat organs or raw flesh for nutrients, also seems to become hungrier at a faster rate"
+	scan_desc = "H_(82882)G3E:__))9R"
+	gain_text = span_warning("I feel a hunger, only organs and flesh will feed it...")
+	lose_text = span_notice("Your stomach no longer craves flesh, and your tongue feels duller.")
+	random_gain = FALSE
+	/// How much faster we loose hunger
+	var/hunger_rate = 15
+
+/datum/brain_trauma/severe/flesh_desire/on_gain()
+	// Allows them to eat faster, mainly for flavor
+	ADD_TRAIT(owner, TRAIT_VORACIOUS, REF(src))
+	// If they have a tongue, make it crave meat
+	var/obj/item/organ/internal/tongue/tongue = owner.get_organ_slot(ORGAN_SLOT_TONGUE)
+	if(tongue)
+		tongue.liked_foodtypes = GORE | MEAT
+	..()
+
+/datum/brain_trauma/severe/flesh_desire/on_life(seconds_per_tick, times_fired)
+	// Causes them to need to eat at 10x the normal rate
+	owner.adjust_nutrition(-hunger_rate * HUNGER_FACTOR)
+	if(SPT_PROB(20, seconds_per_tick))
+		to_chat(owner, span_notice("You feel a ravenous hunger for flesh..."))
+	owner.overeatduration = max(owner.overeatduration - 200 SECONDS, 0)
+
+/datum/brain_trauma/severe/flesh_desire/on_lose()
+	REMOVE_TRAIT(owner, TRAIT_VORACIOUS, REF(src))
+	return ..()
```

Comment:
so we crave meat even after we loose the trauma? what happens if we change tongues?